MonoGame.Framework/Audio/SoundEffectInstance.OpenAL.cs

```diff
@@ -260,14 +308,44 @@ private void PlatformSetVolume(float value)
 
         internal void PlatformSetReverbMix(float mix)
         {
+#if !MONOMAC
+            if (!OpenALSoundController.Efx.IsInitialized)
+                return;
+            reverb = mix;
+#endif
         }
 
         internal void PlatformSetFilter(FilterMode mode, float filterQ, float frequency)
         {
+#if !MONOMAC
+            if (!OpenALSoundController.Efx.IsInitialized)
+                return;
+
+            applyFilter = true;
+            switch (mode) {
+            case FilterMode.BandPass:
+                filterType = EfxFilterType.Bandpass;
+                break;
+                case FilterMode.LowPass:
+                filterType = EfxFilterType.Lowpass;
+                break;
+                case FilterMode.HighPass:
+                filterType = EfxFilterType.Highpass;
+                break;
+            }
+            this.filterQ = filterQ;
+            this.frequency = frequency;
+#endif
```

Comment:
ok so if its NOT playing we need to apply it before we do, but if we are we need to apply it immediately?